src/lib/BaseWSClient.ts

```diff
@@ -530,7 +531,7 @@ export abstract class BaseWebsocketClient<
 
       this.clearPongTimer(wsKey);
 
-      this.getWs(wsKey)?.terminate();
+      safeTerminateWs(this.getWs(wsKey));
```

Comment:
```suggestion
      safeTerminateWs(this.getWs(wsKey), true);
```

This is the _only_ call to close it, so here we do need to fallback to "close" if terminate is not available, or we have no way of closing this socket.